drivers/axi_core/axi_dmac/axi_dmac.c

```diff
@@ -47,6 +47,49 @@
 #include "delay.h"
 #include "axi_dmac.h"
 
+/***************************************************************************//**
+ * @brief dma_isr
+*******************************************************************************/
+void axi_dmac_isr(void *instance)
+{
+	struct axi_dmac *dmac = (struct axi_dmac *)instance;
+	uint32_t user_index = dmac->user_index, driver_index = dmac->driver_index;
```

Comment:
It is easy to read if you add driver_index in a new line.

Reply:
This has changed completely.